Common/UnitEnumValues.g.json

```diff
@@ -1343,7 +1343,8 @@
     "UsSurveyFootPerSecond": 29,
     "YardPerHour": 30,
     "YardPerMinute": 31,
-    "YardPerSecond": 32
+    "YardPerSecond": 32,
+    "Mach": 42
```

Comment:
You asked about why the number is 42 and not 33. This is intentional to reduce merge conflicts, when multiple pull requests adds units to the same quantity. It picks a random value for the next 10 available values.